# The speaker-safe paths enable the speaker instead of the default device.
--- a/etc/mixer_paths_tavil.xml
+++ b/etc/mixer_paths_tavil.xml
@@ -616,7 +616,7 @@
     </path>
 
     <path name="deep-buffer-playback speaker-safe">
-        <path name="deep-buffer-playback" />
+        <path name="deep-buffer-playback speaker" />
     </path>
 
     <path name="deep-buffer-playback hdmi">
@@ -723,7 +723,7 @@
     </path>
 
     <path name="low-latency-playback speaker-safe">
-        <path name="low-latency-playback" />
+        <path name="low-latency-playback speaker" />
     </path>
 
     <path name="low-latency-playback hdmi">
@@ -831,7 +831,7 @@
     </path>
 
     <path name="audio-ull-playback speaker-safe">
-        <path name="audio-ull-playback" />
+        <path name="audio-ull-playback speaker" />
     </path>
 
     <path name="audio-ull-playback headphones">
@@ -927,7 +927,7 @@
     </path>
 
     <path name="compress-offload-playback speaker-safe">
-        <path name="compress-offload-playback" />
+        <path name="compress-offload-playback speaker" />
     </path>
 
     <path name="compress-offload-playback hdmi">
